xml/ftp_banners.xml

```diff
@@ -129,16 +129,27 @@ example.com FTP server (Version:  Mac OS X Server) ready.</example>
     <param pos="2" name="service.version"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
-  <fingerprint pattern="^(\S+) FTP Server \(SunOS 5.([789]|\d{2,})\) ready\.?$" flags="REG_ICASE">
+  <fingerprint pattern="^(\S+) FTP Server \(SunOS 5.(1[1-9]|[2-9]\d{1,}|\d{3,})\) ready\.?$" flags="REG_ICASE">
```

Comment:
Is the increasingly high 2 digit 3+ digit version numbers realistic?  AFAIK, Solaris 123 is not a thing.